_episodes_rmd/05-data-visualization.Rmd

```diff
@@ -35,23 +31,68 @@ knitr_fig_path("05-")
 variants <- read.csv("https://ndownloader.figshare.com/files/14632895")
 ```
 
-We start by loading the package **`ggplot2`**.
+We start by loading the package **`ggplot2`**. 
 
-```{r load-ggplot2, message=FALSE, purl=FALSE}
+```{r load-ggplot2, message=FALSE, purl = TRUE}
 library(ggplot2)
 ```
+
+The *gg* in **`ggplot2`** stands for "grammar of graphics". The idea is that 
+any tidy data set can be made into an informative plot in a systematic way, by mapping
+variables in the data to positions, colours, shapes, and sizes of visual output. 
+These visual features are referred to as "aesthetics", or "aes" for short.
+There are different ways of mapping aesthetics to visual features - 
+bar plots, line plots, and so on. These differ in their geometry and so are referred 
+to as "geometries", or "geom" for short.
+
+Some simple examples will help. We will first plot one of R's built-in datasets,
+the [*Iris* flower dataset](https://en.wikipedia.org/wiki/Iris_flower_data_set),
+which measured sizes of iris flowers in different species, in 1930s Canada^[The iris dataset was collected by scientists who supported eugenics, a discredited and discriminatory historical practice. More background is given on the wikipedia page.].
```

Comment:
There's been a recent push to explicitly NOT include the iris data set in teaching examples for this very reason. Since this example doesn't really focus on any aspects specific to the iris data set, I would suggest to use an alternative data set that is included with R (e.g. mtcars, or sleep), or [a data set included with {ggplot2}](https://ggplot2.tidyverse.org/reference/index.html#data) (e.g. msleep)

Reply:
I agree we'd be better without `iris`. It would be nice to have another dataset where it's easy to have a plot with two numerical values, without any axis transformations, with colours that make sense. I don't see that with `msleep` - bodywt and brainwt are too dispersed, and the categorical variables don't separate the data. The plot is easier with `mpg`, but it's odd to teach about cars to genomics people.

Also, the `variants` dataset is not an intuitive start to learning a plot program. That's why 

Is there any chance we could use [palmerpenguins](https://allisonhorst.github.io/palmerpenguins/)? Just requires:

```
install.packages("palmerpenguins")
library(palmerpenguins)
```

Then
```
ggplot(data = penguins, aes(x = flipper_length_mm, y = bill_length_mm, colour = species)) +
  geom_point()
```

compared to:

```
ggplot(data = penguins, aes(x = species, y = bill_length_mm, colour = flipper_length_mm)) +
  geom_point()
```

Reply:
+1 for palmerpenguins, but I will defer to one of the maintainers on this repo.